cluster/addons/dns/skydns-rc.yaml.base

```diff
@@ -103,9 +101,7 @@ spec:
       - name: healthz
         image: gcr.io/google_containers/exechealthz-amd64:1.1
         resources:
-          # keep request = limit to keep this container in guaranteed class
           limits:
-            cpu: 10m
```

Comment:
As of v1.4 this pod was in the Burstable QoS class even before this patch. QoS is per pod and per container.